webpack/assets/javascripts/react_app/components/HostDetails/Audits/index.js

```diff
@@ -1,143 +1,101 @@
-                                        width: '350px',
-                                        overflowX: 'hidden',
-                                      }}
-                                    >
-                                      {value && (
-                                        <ElipsisWithTooltip>
-                                          <mark
-                                            style={{
-                                              backgroundColor: '#FFC0CB',
-                                            }}
-                                          >
-                                            {value[0]}
-                                          </mark>
-                                        </ElipsisWithTooltip>
-                                      )}
-                                    </DataListCell>,
-                                    <DataListCell
-                                      style={{
-                                        width: '350px',
-                                        overflowX: 'hidden',
-                                      }}
-                                      key={`new-value-${i}`}
-                                    >
-                                      {value && (
-                                        <mark
-                                          style={{
-                                            backgroundColor: '#90EE90',
-                                          }}
-                                        >
-                                          {value[1]}
-                                        </mark>
-                                      )}
-                                    </DataListCell>,
-                                  ]}
-                                />
-                              </DataListItemRow>
-                            </DataListItem>
-                          )
-                        )}
-                    </DataList>
-                  </AccordionContent>
-                </AccordionItem>
-              );
-            return null;
-          })}
-        </Accordion>
+        <SkeletonLoader
+          skeletonProps={{ count: 3 }}
+          status={status}
+          emptyState={
+            <Bullseye>
+              <Title headingLevel="h4"> {__('No Results found')} </Title>
+            </Bullseye>
+          }
+        >
+          <DataList aria-label="recent-audit-table" isCompact>
+            {audits?.map(
+              ({
+                user_name: user,
+                user_id: userID,
+                created_at: timestamp,
+                action,
+                remote_address: ip,
+                id,
+              }) => (
+                <DataListItem aria-labelledby="simple-item1">
+                  <DataListItemRow>
+                    <DataListItemCells
+                      dataListCells={[
+                        <DataListCell key={`id-${action}`}>
```

Comment:
Should this be 
```suggestion
                        <DataListCell key={`action-${id}`}>
```
to be consistent with the other cells and to avoid duplicate keys in case of e.g. two update audits?